### DISCUSSION TITLE
How can I recover my account

### DISCUSSION THREAD

**Discussion by SeersSocial** [community/community#110439] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hello, I would like to know how to recover my GitHub account. I have the password and access to the associated email, but 2FA is preventing me from logging in. It's sending an SMS to an old phone number to which I no longer have access. Thanks for any help!

**Comment by itsPronay** [comment]
Give it a read @SeersSocial 

https://docs.github.com/en/enterprise-server@3.8/authentication/securing-your-account-with-two-factor-authentication-2fa/recovering-your-account-if-you-lose-your-2fa-credentials

**Comment by itsPronay** [comment]
also you lost your recovery code?

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @SeersSocial ! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Discussions category is solely related to conversations around the GitHub product Discussions. I've gone ahead and moved it for you. Good luck!
